.drone.yml

```diff
@@ -41,13 +46,19 @@ steps:
   commands:
   - "echo -e \"unknwon\nreferer\nerrorstring\neror\niam\nwan\" > words_to_ignore.txt"
   - codespell -I words_to_ignore.txt docs/
+  when:
+    paths:
+    - docs/**
   depends_on:
   - initialize
 
 - name: shellcheck
   image: grafana/build-container:1.4.1
   commands:
   - ./bin/grabpl shellcheck
+  when:
+    paths:
+    - "*.sh"
```

Comment:
i think here we might want to put **.sh, since our shell are not only in the root, i might be wrong regarding what shellcheck is doing here.